lib/loaders/typescript.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const loaderFeatures = require('../loader-features');
+const babelLoader = require('./babel');
+
+/**
+ * @param {WebpackConfig} webpackConfig
+ * @return {Array} of loaders to use for TypeScript
+ */
+module.exports = {
+    getLoaders(webpackConfig) {
+        loaderFeatures.ensureLoaderPackagesExist('typescript');
+
+        // some defaults
+        let config = {
+            transpileOnly: false,
+            happyPackMode: false,
+            logInfoToStdOut: false,
+            logLevel: 'info',
+            silent: true,
```

Comment:
You are right! I thought on being more verbose there to have it documented in code but it is right.

> And why silent: true?

Avoiding to be verbose with CLI output. As [show here](https://github.com/TypeStrong/ts-loader/blob/master/README.md#silent-boolean-defaultfalse) it doesn't affect processing of files, only output. For example without `silent` it is always printed messages about which `tsconfig.json` is used. I think it was too much and was making also the output of test not nice looking since it was printing out of context. 

Basically it is more a cosmetic change.